profile/README.md

```diff
@@ -0,0 +1,47 @@
+# Welcome to Pdftools
+
+### 📄 Transforming Documents, One PDF at a Time
+
+At **Pdftools**, we specialize in providing advanced tools and services for working with PDF documents. Our mission is to build scalable, efficient, and cost-effective solutions for automating PDF-related workflows. 
+
+Whether you're a developer looking for easy-to-use SDKs or a business aiming to streamline document management processes, Pdftools has you covered.
+
+---
+
+## 🚀 What We Offer
+
+- **PDF SDKs - Pdftools SDK & Toolbox**: Powerful software development kits to integrate PDF functionality into your applications.
+- **Conversion Service - Desktop & Automation Tool**: Feature-rich desktop applications for PDF management
+- **PDF Web Viewer**: Pixel perfect PDF viewer & editor for your web application.
+---
+
+## 🏆 Our Core Values
+
+- **Quality first**: We ensure the highest level of quality for document handling.
+- **Performance Driven**: Our tools are built for speed and scalability.
+- **Flexibility**: Customize your PDF workflows to fit your unique requirements.
+
+---
+
+## 💻 Get Involved
```

Comment:
As we're not open source and we don't have anything public (to the best of my knowledge), there aren't any external contributors that can contribute to our repositories and to the evolution of our products. 
Not sure how to rephrase it at the moment but I agree that it's important to have those links.